.github/workflows/build.yml

```diff
@@ -51,7 +51,7 @@ jobs:
       - name: Upgrade pip, install dependencies and the package
         run: |
           python -m pip install --upgrade pip setuptools
-          python -m pip install .[dev,publishing]
+          python -m pip install .[dev,publishing,rl]
```

Comment:
for this and the sonarcloud workflow, can you please pre-install the cpu-only torch before this, otherwise it'll install a few hundred MB worth of CUDA stuff first (this only happens by default on linux, where the workflows are running, not on mac or windows).
You can copy what I did in the static typing workflow.